src/modules/couplingXCubHandMk5/CMakeLists.txt

```
@@ -0,0 +1,41 @@
+# Copyright (C) 2006-2023 Istituto Italiano di Tecnologia (IIT)
+# All rights reserved.
+#
+# This software may be modified and distributed under the terms of the
+# BSD-3-Clause license. See the accompanying LICENSE file for details.
+
+yarp_configure_plugins_installation(ergocub-software)
+
+yarp_prepare_plugin(couplingXCubHandMk5
+                    CATEGORY device
+                    TYPE CouplingXCubHandMk5
+                    INCLUDE CouplingXCubHandMk5.h
+                    DEFAULT OFF)
```

```suggestion
                    DEFAULT OFF)
```

There is any reason why this by default is OFF? If I am not wrong, this device does not have any additional dependency on top of YARP (that is already a requirement of the repo).

---

Originally it was off because when I started working on it YARP was an optional dependency, but yes we can make it ON by default!